lib/fibonacci.rb

```diff
@@ -5,8 +5,19 @@
 # ....
 # e.g. 6th fibonacci number is 8
 
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n) with n being the input.
+# Space complexity: O(1)- constant because it take the same amount of space regardless of the input.
```

Comment:
Space complexity is actually dependent on the number of recursive calls.  Each time you make a call to `fibonacci` it has to save the previous call in the stack.